lib/cli/writer.js

```diff
@@ -67,6 +68,13 @@ exports.writeImage = (imagePath, drive, options, onProgress) => {
     return fs.openAsync(drive.raw, 'rs+');
   }).then((driveFileDescriptor) => {
     return imageStream.getFromFilePath(imagePath).then((image) => {
+      if (!image.size.final.estimation && image.size.final.value > drive.size) {
```

Comment:
I guess this should also be refactored to use the `isDriveLargeEnough` method that the GUI drive-selector code uses?
At the very least the `>` in the line above should be changed to `>=`